pkg/client/cluster_maas.go

```diff
@@ -0,0 +1,189 @@
+package client
+
+import (
+	hapitransport "github.com/spectrocloud/hapi/apiutil/transport"
+	"github.com/spectrocloud/hapi/models"
+
+	clusterC "github.com/spectrocloud/hapi/spectrocluster/client/v1alpha1"
+)
+
+func (h *V1alpha1Client) CreateClusterMaas(cluster *models.V1alpha1SpectroMaasClusterEntity) (string, error) {
+	client, err := h.getClusterClient()
+	if err != nil {
+		return "", err
+	}
+
+	params := clusterC.NewV1alpha1SpectroClustersMaasCreateParamsWithContext(h.ctx).WithBody(cluster)
+	success, err := client.V1alpha1SpectroClustersMaasCreate(params)
+	if err != nil {
+		return "", err
+	}
+
+	return *success.Payload.UID, nil
+}
+
+// TODO: no update for maas?
+/*func (h *V1alpha1Client) UpdateClusterMaas(cluster *models.V1alpha1SpectroMaasClusterEntity) error {
```

Comment:
Update support?

Reply:
As per @rishi-anand, we don't have update support now as the code got refactored. So I'm removing it.

Reply:
Removed.